python/cudf/cudf/io/parquet.py

```diff
@@ -666,7 +738,12 @@ def to_parquet(
             row_group_size_rows=row_group_size_rows,
             max_page_size_bytes=max_page_size_bytes,
             max_page_size_rows=max_page_size_rows,
-            **kwargs,
+            partitions_info=list(
+                zip(partition_offsets, np.diff(partition_offsets))
```

Comment:
Can we revisit this and apply @wence-'s suggestion:

https://github.com/rapidsai/cudf/pull/12025#discussion_r1009484064

There are two places in this file where NumPy is being used, and these can both be replaced by `[(i, j - i) for i, j in zip(offsets, offsets[1:])]` to eliminate the dependence on `import numpy as np`.

Reply:
Apologies @wence- I couldn't see this comment earlier. Now it should be resolved fully.